esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6100,14 +6081,19 @@ bool CWsDfuEx::onDFUFileAccess(IEspContext &context, IEspDFUFileAccessRequest &r
 {
     try
     {
-        SecAccessFlags accessType = translateToSecAccessFlags(req.getAccessType());
+        IConstDFUFileAccessRequestBase &requests = req.getRequestBase();
+        SecAccessFlags accessType = translateToSecAccessFlags(requests.getAccessType());
         if (SecAccess_None == accessType)
-            throw MakeStringException(ECLWATCH_DFU_ACCESS_DENIED, "onDFUFileAccess - Permission denied.");
-        else if (!context.validateFeatureAccess(FEATURE_URL, accessType, false))
-            throw MakeStringException(ECLWATCH_DFU_ACCESS_DENIED, "onDFUFileAccess - Permission denied.");
+        {
+            context.setAuthStatus(AUTH_STATUS_NOACCESS);
```

Comment:
just curious what does this actually do?

Reply:
This will add 'authorization failure' flag into the TxSummary in ESP log.